bom/runtime/pom.xml

```diff
@@ -1284,7 +1301,7 @@
             </dependency>
             <dependency>
                 <groupId>org.testcontainers</groupId>
-                <artifactId>junit-jupiter</artifactId>
+                <artifactId>postgresql</artifactId>
```

Comment:
I had that because vault's tests require the postgresql module from the testcontainers project, and I lacked visibility on property `test-containers.version`.
I reverted my change to `bom/runtime/pom.xml`, and added an explicit version for that dependency in module `quarkus-vault-test` (with the risk that we change the version of testcontainers:org.testcontainers in the bom, and forget to change the version in quarkus-vault-test's pom).
is there a better way to do that?

> BTW, I'm not totally sure it's a good idea to rely on testcontainers for the PostgreSQL part.

it looks like the postgresql container is launched only for native tests. right? whereas I need it for all IT tests (even the non native ones). 
using test containers allows be to launch those tests from the ide, like it was regular unit tests. 
inspired by the good [talk](https://www.youtube.com/watch?v=0TvWv4L_IJM) from vincent massol.
and this allows to have everything in one place container config, init script, username, network bridge, ...